packages/forms-web-app/src/controllers/appellant-submission/supporting-documents.js

```diff
@@ -92,6 +92,7 @@ exports.postSupportingDocuments = async (req, res) => {
 			console.log(appeal);
```

Comment:
I know this is not part of your PR but if we could get it removed pls. Thank you